src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -99,15 +100,26 @@ public static Email parseEmail(String email) throws ParseException {
 
     /**
      * Parses a {@code String payment} into an {@code Payment}.
-     * Leading and trailing whitespaces will be trimmed
+     * Leading and trailing whitespaces will be trimmed, all characters are converted to lower case
      */
     public static Payment parsePayment(String payment) throws ParseException {
         String trimmedPayment = payment.trim().toLowerCase();
```

Comment:
I was thinking if we need this toLowerCase() anymore, maybe we should remove it for clarity?

Reply:
Yeap I will leave it out